File: dynamic_listings/python/default/computer_science/operators.py
# Operators can be treated as functions, that means you can
# Assign them to a variable.

# This...

if a == b and c == d:
    # Do something...
    ...

# Is equivalent to this

complex_condition: bool = (a == b and c == d)

if complex_condition:
    # ...
    ...

# ---------------8<---------------

# Also this...

def thing(a: int, b: int) -> bool:
    if a == b:
        return True
    else:
        return False

# Is equivalent to...

def thing(a: int, b: int) -> bool:
    return a == b

File: dynamic_listings/javascript/default/computer_science/operators.js
// Operators can be treated as functions, that means you can
// Assign them to a variable.

// This...

if (a == b && c == d){
    // Do something...
}

// Is equivalent to this

let complex_condition = (a == b && c == d);

if (complex_condition){
    // ...
}

// ---------------8<---------------

// Also this...

function thing(a, b){
    if (a == b){
        return true;
    }else{
        return false;
    }
}

// Is equivalent to...

function thing(a, b){
    return a == b;
}
